earthpy/tests/test_plotrgb.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for plot_rgb function"""
+
+import numpy as np
+import numpy.ma as ma
+import matplotlib.pyplot as plt
+import pytest
+import rasterio as rio
+from earthpy.plot import plot_rgb
+from earthpy.io import path_to_example
+
+plt.show = lambda: None
```

Comment:
this is a good question. this seemed to make travis happy and it is what rasterio uses -- see here

https://github.com/mapbox/rasterio/blob/master/tests/test_plot.py 

when i didn't have that i got these travis errors because i think it was trying to plot things but had no frame to render on. i couldn't get it to work otherwise. haven't revisited it for a while but i spent a good bit of time on it and it seemed necessary at the time.Note this wasn't an issue locally it was an issue specific to travis.

Reply:
Cool - thanks!